awx/ui_next/src/screens/Template/JobTemplateDetail/JobTemplateDetail.jsx

```diff
@@ -245,6 +249,35 @@ function JobTemplateDetail({ i18n, template }) {
             />
           </React.Fragment>
         )}
+        {webhook_service && (
+          <Detail
+            label={i18n._(t`Webhook Service`)}
+            value={
+              webhook_service === 'github'
+                ? i18n._(t`GitHub`)
+                : i18n._(t`GitLab`)
+            }
+          />
+        )}
+        {webhook_receiver && (
+          <Detail
+            label={i18n._(t`Webhook URL`)}
+            value={`${document.location.origin}${webhook_receiver}`}
+          />
+        )}
+        <Detail label={i18n._(t`Webhook Key`)} value={webhook_key} />
```

Comment:
should we be hiding this if we don't receive it in {webhook_key && ...} like the other fields?

Reply:
DetailValue will hide it automatically if there is no value. https://github.com/ansible/awx/blob/devel/awx/ui_next/src/components/DetailList/Detail.jsx#L43